data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -343,3 +342,46 @@
 AND value_as_string != 'PMI Skip'
 '''
 pd.read_gbq(query, dialect='standard')
+
+# Check pid_rid_mapping table for duplicates
+query = f'''
+SELECT 'person_id' as id_type, person_id as id, COUNT(person_id) as count
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+GROUP BY person_id
+HAVING count > 1
+UNION ALL
+SELECT 'research_id' as id_type, research_id as id, COUNT(research_id) as count
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+GROUP BY research_id
+HAVING count > 1
+'''
+pd.read_gbq(query, dialect='standard')
+
+# Ensure all person_ids exist in the person table
+query = f'''
+SELECT 'missing_person' as issue_type, person_id
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+WHERE person_id NOT IN 
+(SELECT person_id
+FROM `{project_id}.{new_rdr}.person`)
+UNION ALL 
+SELECT 'unmapped_person' as issue_type, person_id
+FROM `{project_id}.{new_rdr}.person`
+WHERE person_id NOT IN 
+(SELECT person_id
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`)
+'''
+pd.read_gbq(query, dialect='standard')
+
+# Check for inconsistencies between primary and rdr pid_rid_mappings
+# Mappings which were in previous exports may be removed from a new export for two reasons:
+#   1. Participants have withdrawn or
+#   2. They were identified as test or dummy data
```

Comment:
I would add how to interpret the result set to this description.
1. What does it look like when no problems are indicated.
2. What does it look like when problems are indicated.
3. Who to contact when problems are indicated.  I think this would require coordinating with the RDR team to determine what happened with the mapping.